libyear/export_format/__init__.py

```diff
@@ -0,0 +1 @@
+name = "libyear"
```

Comment:
What is this for? ^^

Reply:
copy/paste from another __init__ file (so lazy with them I just copy past one to another without paying attention :D)